freqtrade/data/btanalysis.py

```diff
@@ -81,19 +81,30 @@ def load_trades_from_db(db_url: str) -> pd.DataFrame:
     """
     trades: pd.DataFrame = pd.DataFrame([], columns=BT_DATA_COLUMNS)
     persistence.init(db_url, clean_open_orders=False)
-    columns = ["pair", "profit", "open_time", "close_time",
-               "open_rate", "close_rate", "duration", "sell_reason",
-               "max_rate", "min_rate"]
 
-    trades = pd.DataFrame([(t.pair, t.calc_profit(),
+    columns = ["pair", "open_time", "close_time", "profit", "profitperc",
+               "open_rate", "close_rate", "amount", "duration", "sell_reason",
+               "fee_open", "fee_close", "open_rate_requested", "close_rate_requested",
+               "stake_amount", "max_rate", "min_rate", "id", "exchange",
+               "stop_loss", "initial_stop_loss", "strategy", "ticker_interval"]
+
+    trades = pd.DataFrame([(t.pair,
                             t.open_date.replace(tzinfo=pytz.UTC),
                             t.close_date.replace(tzinfo=pytz.UTC) if t.close_date else None,
-                            t.open_rate, t.close_rate,
+                            t.calc_profit(), t.calc_profit_percent(),
+                            t.open_rate, t.close_rate, t.amount,
                             t.close_date.timestamp() - t.open_date.timestamp()
                             if t.close_date else None,
```

Comment:
indentation should be applied here?

Reply:
I thought so / think so  - but flake disagrees ... but it works with parenteses ...

Reply:
👍